package.json

```diff
@@ -31,17 +31,18 @@
       "scripts/create-component/package-template/**/*"
     ],
     "linters": {
+      "**/*.css": [
+        "prettier-eslint --write",
+        "git add"
+      ],
       "**/*.js": [
         "prettier-eslint --write",
+        "eslint",
```

Comment:
i was confused. `prettier-eslint` just formats the file. we can use `eslint` to do the linting